packages/components/pages/f-self-exclusion/package.json

```diff
@@ -48,20 +48,20 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-globalisation": "1.2.0",
-    "@justeat/f-services": "1.x",
     "@justeat/f-alert": "6.x",
     "@justeat/f-button": "5.x",
     "@justeat/f-card": "4.x",
     "@justeat/f-form-field": "6.x",
+    "@justeat/f-globalisation": "1.2.0",
```

Comment:
Maybe worth modifying this to `1.x` for consistency. It should still resolve the same version, just means the component doesn't need updating for any future minor versions of this package.